readme.md

```diff
@@ -20,45 +20,62 @@ pdf2json is a [node.js](http://nodejs.org/) module converts binary PDF to JSON a
 
 ## Install
 
-> npm i pdf2json
+You can install it using npm or bun:
 
-Or, install it globally:
+```bash
+npm install pdf2json
+bun add pdf2json
+```
 
-> npm i pdf2json -g
+If you want to use the `pdf2json` CLI, you can install it globally:
 
-To update with latest version:
+```bash
+npm install pdf2json -g
+bun install pdf2json -g
+```
 
-> npm update pdf2json -g
+## Usage
 
-To Run in RESTful Web Service or as command line Utility
+```javascript
+import PDFParser from "pdf2json";
 
-- More details can be found at the bottom of this document.
+const pdfParser = new PDFParser();
+```
 
-## Test
+The module is tested with [Node.js](https://nodejs.org/) 18+ and [Bun](https://bun.sh/) 1+.
 
-After install, run command line:
+## Test
 
-> npm test
+You can run tests in Bun, or in Node.js using Jest.
 
-`pretest` step builds bundles and source maps for both ES Module and CommonJS, output to `./dist` directory. The Jest test suit is defined in `./test/_test_.cjs` with commonJS, test run will also cover `parse-r` and `parse-fd` with ES Modules via command line.
+```bash
+bun run test:bun # runs in Bun
+bun run test:node # runs in Node.js using Jest
```

Comment:
better keep `npm test` as to not breaking existing devop automation.
keep the `bun` related info as extension, not replacement.